utreexo/pollardutil.go

```diff
@@ -5,6 +5,7 @@ import (
 	"os"
 
 	"github.com/mit-dci/utreexo/cmd/util"
+	"github.com/mit-dci/utreexo/log"
```

Comment:
Did you add a package to the repo? In the commit, there is no log/ directory

Reply:
Oh, right, that explains the failure too. Sorry, dunno how I could have missed this.